client/src/store/actions/authActions.tsx

```diff
@@ -8,12 +8,13 @@ import {
   RegisterActions,
   LoginData,
   RegisterData,
-  LoadUserActions
+  LoadUserActions,
+  LogoutAction
 } from "../interfaces/authTypes";
 
 type ErrorResponse = {
-  errors: string
-}
+  errors: string;
```

Comment:
Are you sure that in object properties semicolons are using?

Remove this semicolor, please

Reply:
It is not an object but type definition.